README.md

```diff
@@ -68,9 +78,9 @@ Select all that apply:
 ```
 [] `$(".post").css("background", "peachpuff")`
```

Comment:
This is correct.  Using just "background" as the attribute is a bit tricky.  I recommend you try the code in the console.  Just make sure you understand why something works, or does't.